### DISCUSSION TITLE
Edit the Cookie Banner?

### DISCUSSION THREAD

**Discussion by localghost8000** [studio1902/statamic-peak#327] [Q&A] [answered]
Apologies if this has been asked before, but I'm trying to utilize the cookie banner that is configured in the SEO settings. The banner loads perfectly, but I want to edit the visuals. Is there a way to publish this component?

**Comment by robdekort** [comment, marked as answer]
Yep! After this trash the files you don’t wanna make changes to. 

https://peak.1902.studio/features/social-images-generation.html#installation-and-configuration
